### DISCUSSION TITLE
Swiper in loop mode is moving really fast in reverse direction to first slide

### DISCUSSION THREAD

**Discussion by phifa** [nolimits4web/swiper#4591] [Q&A]
In loop mode, sometimes, when the last image is reached, instead of continuing to the right it goes all the way to the left, it rewinds back to the first slide. Anyone ever had that and knows how to properly set this up? I am sure this is a config thing.

**Comment by twobit-hack** [comment]
I hope you found your solution, but for those like me who found this unanswered question, here is what fixed it for me:
`initialSlide: 5,`
setting an initial slide other than the first one stops the carousel from trying to loop quickly in reverse. i hope this helps somebody!
